packages/riipen-ui/src/components/Editor.jsx

```diff
@@ -256,8 +256,9 @@ class Editor extends React.Component {
         font-size: ${theme.typography.body1.fontSize};
 
         /* 80px - 2 * 1px borders  */
+        max-height: 300px;
         min-height: 78px;
-        overflow: auto;
+        overflow-y: scroll;
```

Comment:
I _think_ if you have a `max-height` you actually can leave this as `overflow-y: auto` which will only show the scroll bar if it is needed. A value of `scroll` will _always_ show the bar even if it's not needed. Can you test that and confirm?

Reply:
The scroll bar shows up with a given max height and `overflow-y: auto`